app/styles/styles.css

```diff
@@ -0,0 +1,4 @@
+.loginForm {
+	width: 50%;
+	margin: 0 auto;
```

Comment:
Super-minor: missing indentation.

Reply:
It should be 2 spaces right? I'll make the change.

Reply:
Yup, thank you!